### ISSUE TITLE
Dimension error on Z-axis-bottom-right and -left

### ISSUE THREAD

**Issue by RH-Dreambox** [prusa3d/Prusa-i3-Plus#49]
Hi,
I have found a dimension error on the two parts Z-axis-bottom-right and -left that should be fixed.
The console is 0.5 mm too narrow and the Z-motors can not be mounted centered in the intended location.

The Z-screws will therefore lean relative to the Z-smooth-rod, which may cause operating problems.
Please, read a detailed description in my post "Fine tune your MK3 Z axis.

[https://shop.prusa3d.com/forum/original-prusa-i3-mk3-f30/fine-tune-your-mk3-z-axis-t13806.html](url)

**Comment by gregsaun** [comment]
Thank you for finding and sharing this, I could reproduce it as well on my side with MK2s and MK3. Here is a view with motor aligned on screws : 

![motor_mount_issue](https://user-images.githubusercontent.com/12335098/35441937-b3c34fde-02a5-11e8-8dad-b4861b28b55c.png)

**Comment by stahlfabrik** [comment]
Was this issue resolved with the R2 parts?

**Comment by RH-Dreambox** [comment]
No, I had to fix that on my own details.

**Comment by stahlfabrik** [comment]
Are there modified R2 parts that JUST include a correction for that issue?

**Comment by c-stewart** [comment]
I've only seen corrected versions that also include the extra slot for
quick leadacrew removal. Made by both of the other people posting in this
thread actually.

**Comment by stahlfabrik** [comment]
Yes (but thanks!), I am aware of two version (JTLX and Gregsaun). But these are not just "pure" fixes. They change a lot and optimize for PETG. I would like to have an official fixed (and why not also improved) version, that is PETG or ABS ready...:-)

By the way: Again, I cannot understand why nobody from Prusa Research is participating here. It would cost hardly anything to comment on open issues. On Slic3r and also firmware that is working quite fine now. The hardware (printed parts!) issues are generally not commented by Prusa Research - which makes me sad - as there is so much potential in communicating with the community in an open source project. I know it is obvious - sorry for the short rant.

**Comment by c-stewart** [comment]
Indeed, I think you stated that clearly in your other message. I was just
answering 'no' to your question :)

**Comment by c-stewart** [comment]
You could try tagging some of the developers in the message stream here to
try to get their attention and make a new part. I'm not sure who's in
charge of printed parts though.

**Comment by c-stewart** [comment]
Or if someone can alter the r2 part with just the fix they could create a
pull request.

**Comment by RH-Dreambox** [comment]
Here is my parts
[www.swedenroots.se/download/My_STL_parts.ZIP](url)

**Comment by stahlfabrik** [comment]
Can somebody with the right understanding of the issue please create a Pull Request -- or any word from Prusa Research if that is a valid issue and how to patch which scad file?

**Comment by triplepoint** [comment]
Just to add to the conversation, I think this i why we don't see pull requests accepted on this repository:
https://shop.prusa3d.com/forum/hardware-firmware-and-software-help-f64/3d-model-release-plan-and-the-parts-git-repo-in-ge-t23803.html
